server/src/index.js

```diff
@@ -46,6 +47,10 @@ async function start() {
   applyFileMiddelware(app);
   app.get('/health', (req, res) => res.sendStatus(200))
 
+  app.use('/graphql', altairExpress({
+    endpointURL: '/graphql'
```

Comment:
Is there a way to pass queries that we have defined in `config.playgroundConfig`? 

See `apolloConfig.playground` (we should probably disable that as well)